### DISCUSSION TITLE
Does whisper output emojis?

### DISCUSSION THREAD

**Discussion by arnavmehta7** [openai/whisper#1205] [Q&A] [answered]
I was transcribing a audio and it had symbols of music in it. I was wondering why does whisper outputs emojis? ```🎵``` was the emoji in output.

**Comment by EtienneAb3d** [comment]
Certainly because Whisper was trained with video subtitles containing such emojis. It does what it was trained to do according to the data used.

**Reply by RuslanZalikov** [reply]
Hello, how can I make it not return emoji

**Reply by arnavmehta7** [reply, marked as answer]
Yes whisper output's emojis. But you can use suppress method inside code to suppress them.
